repo2docker/app.py

```diff
@@ -245,7 +247,8 @@ def get_argparser(self):
 
         argparser.add_argument(
             '--ref',
-            help='If building a git url, which ref to check out'
+            help=('If building a git url, which ref to check out. Must be of'
```

Comment:
is ref a jargon-y word for reference? I am not sure if ref is the shorthand for reference or not. If it is shorthand we should make the effort to spell it out.